graylog2-web-interface/src/components/common/ExpandableListItem.tsx

```diff
@@ -208,9 +210,9 @@ class ExpandableListItem extends React.Component<Props, State> {
     }
   };
 
-  _filterInputProps = (props) => {
+  _filterInputProps = () => {
```

Comment:
Not sure why this part was changed too. I think we should keep the changes to fixing the UI issue.